modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -55,14 +56,21 @@ export interface ParsedSignatureScript2Of3 extends ParsedSignatureScript {
   pubScript: Buffer;
 }
 
-export function getDefaultSigHash(network: Network): number {
+export interface ParsedSignatureScriptTaproot extends ParsedSignatureScript {
+  // P2TR tapscript spends are for 2-of-2 multisig scripts
+  signatures: [Buffer | 0, Buffer | 0]; // missing signature is encoded as OP_0
```

Comment:
@sangaman ran into a situation where missing signature was encoded as `Buffer.of()` instead of `0`.

Reply:
Don't think the difference matters here, come to think of it.

Reply:
I think `Buffer.of()` is the correct encoding